### ISSUE TITLE
Normalization process for released FlowNetS model

### ISSUE THREAD

**Issue by yangwangx** [ClementPinard/FlowNetTorch#15]
In README, you mentioned

> pretrained models however (with 'from_scratch' appended to the name) expects a normalized inputs as in the training code.

From what I understand, the normalization should be as follows.   
(1) RGB / 255.0  
(2) normalize with (mean=[0.485, 0.456, 0.406], std=[0.229, 0.224, 0.225])  

Is this correct ? If not, could you tell me about the normalization process or where I should look into the code? 
Thank you.

**Comment by ClementPinard** [comment]
That is correct !
